docs/update_bios.rst

```diff
@@ -10,8 +10,8 @@ instructions will vary depending on the manufacturer and model of your device.
 What you need
 ~~~~~~~~~~~~~
 
-  #. A clean USB device to download the ``.bio`` file
-  #. An internet-connected workstation, such as the *Admin Workstation*
+  #. A clean USB device to download the BIOS file
```

Comment:
As noted in #187, I would recommend phasing out this chapter of the documentation; I can do that in a separate follow-up PR. For now, the changes in this document should cover us for the supported NUC models.